test/cql-pytest/test_secondary_index.py

```diff
@@ -287,6 +287,23 @@ def test_too_large_indexed_collection_value(cql, test_keyspace):
         with pytest.raises(InvalidRequest, match='size'):
             cql.execute(f"INSERT INTO {table}(p,c,m) VALUES (0,1,{{'{big}': 'hi'}})")
 
+# Reproduces #13548:
+# Create a huge index on a table and immediately recreate the table (drop and create)
+# With `--enable-keyspace-column-family-metrics 1` cmd line param, this should slow down the view builder
+# (which is busy building the index), enough to get rid of the first table object from the `database` object
+# but also allow for creating another table with the same name, which raises a double-metrics-registration exception
+@pytest.mark.xfail(reason="issue #13548")
+def test_too_large_indexed_value_build(cql, test_keyspace):
```

Comment:
Interesting that the only way we know to reproduce this bug is to use a too-large (larger than 64KB) view key in this test. This doesn't just "slow down the view builder" (you write smaller items just to give the view builder work) - we actually have a bug in this area - as I explained in https://github.com/scylladb/scylladb/issues/8627, Scylla should just skip this oversized value while building the index (since it can't be indexed) but today's code goes into a retry loop in the view building phase. Maybe the artificial sleeps in that (buggy) code path is the only thing that guarantees reproducing this bug? But to be honest, I'm not sure why a slow or buggy view build even effects the issue of DROPing a table - it doesn't wait for the view build to finish, as far as I know...
Anyway, relying on that bug in a test is unfortunate, it means this reproducer will stop reproducing if we ever fix #8627 - but I guess it's good enough for the purpose of reproducing #13548 before #8627 is fixed.

Reply:
On one hand, this additional test doesn't make a lot of sense, since `test_secondary_index.py` suite already reproduces the issue in 2 places (just not in a single test, rather in between the tests), and it's sole purpose was to present a minimum example to reproduce the issue I'm trying to solve. On the other hand, the test suite can change and wouldn't test the bug, and even if the #8627 is solved, it probably wouldn't hurt anybody to leave this test here.

Reply:
On second reading, I don't understand something... We already have a test with the name test_too_large_indexed_value_build, which it seems you copied and added a single line at the end. Which is fine (even if they are similar tests, they can help reproduce a different bug when run individually - so it's fine to have both), but I don't understand how things work when you have two tests with the same name?

Reply:
The tests shouldn't have the same name. I just copy-pasted the test without actually renaming it - my bad, sorry. On my private setup, I have all other tests commented out, that's why it works for me. 
Next time I'll do my best to polish the draft PR, I assumed it will not be precisely reviewed, but we'd only discuss the general design.

Reply:
You could inject a sleep on the right path in the view building code instead. The advantage of error injection is that it won't burn cpu cycles when run in CI